Core/Source/MVVM/ViewModel.swift

```diff
@@ -65,12 +65,26 @@ public struct SecondaryActionInfo {
     public let imageName: String?
 }
 
+/// Describes the parent item of a list of SecondaryActions.
+public struct SubactionsRootInfo {
```

Comment:
What do you think about either:
a) reusing `SecondaryActionInfo` (documenting the `action` part is ignored in the subaction setting
b) renaming this slightly and making it a substruct of `SecondaryActionInfo` like. Then we're actually sharing the same set of metadata between actions and any subactions.

```
public struct SecondaryActionInfo {
  public struct Metadata {
    public let title: String
    public let state: State
    public let enabled: Bool
    public let imageName: String?
  }

  public let metadata: Metadata

  // for back compat do dynamic getters
  public var title: String {
    return metadata.title
  }
```

Reply:
fwiw: `SubactionsRootInfo` also makes less sense since these can be nested deeply